scripts/cibuild/ConfigureVstsBuild.ps1

```diff
@@ -84,6 +84,17 @@ $regKeyNuGet32 = "HKLM:SOFTWARE\Wow6432Node\Microsoft\StrongName\Verification\*,
 
 
 $has32bitNode = Test-Path "HKLM:SOFTWARE\Wow6432Node"
+
+# update submodule NuGet.Build.Localization
+$NuGetClientRoot = $env:BUILD_REPOSITORY_LOCALPATH
+$Submodules = Join-Path $NuGetClientRoot submodules -Resolve
+
+$NuGetLocalization = Join-Path $Submodules NuGet.Build.Localization -Resolve
+$updateOpts = 'pull', 'origin', 'master'
+           
+Write-Host "git update NuGet.Build.Localization"
+& git -C $NuGetLocalization $updateOpts
```

Comment:
is git available on the path on the build machine? I don't see the path to git.exe anywhere.